src/converters/toZigbee.ts

```diff
@@ -4566,3 +4566,47 @@ export const TS110E_light_onoff_brightness: Tz.Converter = {
         return await light_onoff_brightness.convertSet(entity, key, value, meta);
     },
 };
+
+export const owonFds315SetFallSettings: Tz.Converter = {
```

Comment:
Please move both the fromZigbee and toZigbee changes into `owon.ts` under a `tzLocal`/`fzLocal`, example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/0841340f43ccad8dab8bef23015bcec7aa93c4f0/src/devices/dlink.ts#L8